Makefile

```diff
@@ -174,6 +174,7 @@ e2e-api-test:
 e2e-test:
 	# Run e2e test
 	ginkgo -v ./test/e2e-test
+	ginkgo -v ./e2e/plugin
```

Comment:
please remove this line, the related tests of plugin are implemented in e2e-api-test